### DISCUSSION TITLE
Python not found help pls

### DISCUSSION THREAD

**Discussion by DeltaZuluOG** [atom/atom#22844] [Q&A]
I try to code in python but when i run the code it says Python not found. I tried reinstalling Python and it won't work. Any help?
![Screenshot 2021-08-10 202329](https://user-images.githubusercontent.com/88754077/128955479-cd5d18db-0518-4280-a97c-cd01af031ce6.png)

**Comment by icecream17** [comment]
can you try it in cmd and see if the same error pops up?

in cmd type "python" press enter
